src/main/java/org/jabref/gui/groups/GroupDialog.fxml

```diff
@@ -99,6 +100,12 @@
                             <Tooltip text="%Group containing entries cited in a given TeX file"/>
                         </tooltip>
                     </RadioButton>
+                    <RadioButton fx:id="popularityButton" toggleGroup="$type" wrapText="true"
+                                 text="%Popularity">
+                        <tooltip>
+                            <Tooltip text="%Group containing entries cited in a given TeX file"/>
+                        </tooltip>
+                    </RadioButton>
```

Comment:
This is another functionality. CHANGELOG only talks about view count. Did you work on another issue, too?